lib/lightning_web/channels/attempt_channel.ex

```diff
@@ -49,7 +51,12 @@ defmodule LightningWeb.AttemptChannel do
 
   @impl true
   def handle_in("fetch:attempt", _, socket) do
-    {:reply, {:ok, AttemptJson.render(socket.assigns.attempt)}, socket}
+    {:reply,
+     {:ok,
+      AttemptJson.render(socket.assigns.attempt,
```

Comment:
@stuartc , is this how we expect to send an attribute like `include_run_results` to the worker in the plan? I'd expect this to be next to other attributes like `allow_console_logging`, etc.